packages/react/src/provider.tsx

```diff
@@ -20,6 +20,8 @@ export const LogtoProvider = ({ logtoConfig, children }: LogtoProviderProperties
           setIsAuthenticated(client.isAuthenticated());
         },
       });
+      setIsAuthenticated(client.isAuthenticated());
+      // Help me: need to ensure that executes after `setIsAuthenticated`
       setLogtoClient(client);
```

Comment:
didn't read the context, but intuitively, either use 2 `useEffect()` (less pretty) either re-design the data flow. @simeng-li for more context

Reply:
outdated, thouse code has been removed